docs/book/src/topics/multitenancy.md

```diff
@@ -0,0 +1,74 @@
+# Multi-tenancy
+
+To enable single controller multi-tenancy, a different Identity can be added to the Azure Cluster that will be used as the Azure Identity when creating Azure resources related to that cluster.
+
+This is achieved using the [aad-pod-identity](https://azure.github.io/aad-pod-identity) library. 
+
+## Service Principal Identity
+
+Once a new SP Identity is created in Azure, the coresponding values should be used to create an `AzureClusterIdentity` resource
+
+```yaml
+apiVersion: infrastructure.cluster.x-k8s.io/v1alpha3
+kind: AzureClusterIdentity
+metadata:
+  name: example-identity
+  namespace: default
+spec:
+  type: ServicePrincipal
+  tenantID: <azure-tenant-id>
+  clientID: <client-id-of-SP-identity>
+  clientSecret: {"name":"<secret-name-for-client-password>","namespace":"default"}
+```
+The password will need to be added in a secret similar to the following example
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: <secret-name-for-client-password>
+type: Opaque
+data:
+  clientSecret: <client-secret-of-SP-identity>
+```
+
+OR the password can also as a Certificate 
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: <secret-name-for-client-password>
+type: Opaque
+data:
+  certificate: CERTIFICATE
+  password: PASSWORD
```

Comment:
this is unclear to me? is it one or the other?

Reply:
both kinds are supported by aad-pod-identity https://github.com/Azure/aad-pod-identity/tree/master/deploy/demo